### ISSUE TITLE
[REQUEST] Templar Weapon Change to Maces? (Thoughts on what would be a VERY small change overall?)

### ISSUE THREAD

**Issue by WindowTalents** [Blackstone-SS13/BLACKSTONE#1400]
## Is your feature request related to a problem? Please describe. 
I see Templars do a LOTTA fragging, when they really shouldn't actually be round ejecting players - in the Medbay.

If somebody smacks the Priest, they should have all the validation they need to viciously rip that guy to shreds - but we all know they just stand around waiting to kill literally anybody for sneezing in the Church wrong. Which, is dumb. (If they are even IN the Church that is.)

## Describe the solution you'd like

Good Templars exist and I do not want THEM punished, soooo what about giving the Templars a significantly "less lethal" weapon choice?

What about arming Templars with Maces/Flails instead of the sword? The mace can still kill you, but ONLY with bleeds as there is no instant death crit, and they obviously cannot decapitate people with just their mace; but they are FAR less lethal than swords on every single intent. You can give a valid hungry Templar a mace and he can crush the fucking skulls and break the spines of the towners in the Church BUT that will actually make it less likely for him to *round eject* said towner he beat the fucking shit out of for picking up the crackers in that one chest.

...and if a Templar pulls out a knife to remove a head, that's genuinely on them because they better have a fucking reason for THAT lol.

As it stands, the sword is just killing their targets with bleeds from ten thousand stabs before the medbay players who were just nerfed recently can do anything about it. So I was thinking it could be neat to basically change the Templars kit to let them use the variety of maces we have... so they just don't actually KILL players as often. Once the bleeds from crippling a player is stopped, they are safe to haul around for whatever reason. You literally need them to sleep for a long time OR use Miracles on them to undo this rediclous super stun.

I do not want them to beat people up less, I want them to remain a very important defender of the Church, AND I want them to make people feel uneasy unless they are working for the Church... but yeah bros they usually just slit throats and stab heads like 50 times before there's literally anything anybody can do with or without Miracles. (They just end up wasting devotion and everyone's time.)
## Describe alternatives you've considered

**2 slots instead of 3 for current lowpop.**

I want these dudes to be the "Priest's Knights" and I almost *never* see them behave like that, it is really sad.
I think the issue lies in the fact that most people do not kill the Priest, so they really don't feel like they have anything to do.

I mean like shit dude, OOCly people don't wanna kill the Priest because that fucking ruins the round for the rest of it with stupid ass UNFIXABLE ISSUES that never stop and repeat over and fucking over... We really need a way to replace the Priest... or stop those lol.

(Make the Priest slot open back up after he is fucking dead, why isn't this a thing lol?)
## Additional context

This is going to *MOST LIKELY* mean that _FEMPLARS_ will need to be given a Strength buff so they can wield all the maces.
That is going to mean they will be _literally more powerful than the male ones_, as they always are due to the inherent nature of SS13's "combat" being completely determined by how fast your silly little sprite can move lol.

This doesn't bother me, I just find it very funny that - yeah no women are objectively stronger than men in this game's balance lol.

## TLDR for the Brainrotten: 
Templars should have MACES cause SMASHING PEOPLE WITH GOD'S FUCKING HAMMER is so much cooler than a dumbass sword anyways lol.

I wanna see a Templar paralyze somebody and say "FILTH" to them, and not ROUND EJECT THEM, so they can actually hear it and try to beg for stupid ass Zizo to help them lol.

I think we call that "roleplaying" (™) and you can't do that when you get instantly silenced and sent to the DYING screen from three aimed neck chops with a bastard sword lol.

I'm just saying guys, you can literally survive your throat being slashed in real fucking life. People DRIVE to the ER with slashed throats lol.
OBVIOUSLY, these scenarios are still very fucking rare, but... *you CANNOT survive your SKULL NECK AND SPINE BEING BROKEN ALL AT ONCE like you CAN ON BLACKSTONE LOL!* If the goal of the damage is "realism" this has strayed even further from realism than before.

Honestly, why the fuck is the throat slash SO much more fatal than EVERYTHING ELSE?
I think it causes more bleed than HEART PIERCE wtf? It SHOULD cause oxyloss and bloodloss in a mixed sorta way lmfao.

## Post thoughts below before I start making changes and everybody lynch mobs me lol.
This is NOT a salty cope PR, unless you count "I have ahelped more RDMing Templars than any other role on the entire SERVER" as salt.
I'd love if Templars had maces and legendary mace skill, imagine them just BONKING the SIN outta people doods... imagine...

I'd especially, love, to hear from any actual Templars players who would agree in any form that they want maces instead of swords.
...or the opposite, of course. Please let me know if you think this is dumb as fuck and should be BINNED lol.

**Comment by WindowTalents** [comment]
### Additional Information:

Basically, maces and blunt weapons apply bleeds on the actual hit, but NOT the crits (which is dumb).

A sword will be applying bleeds on every hit, and with every crit it gives you mega bleeds.

A mace smashes your skull and apparently that doesn't make you bleed, but the actual hit does?
Stupid.

The *actual* real true "spine" of this idea is that... I dunno man, I think it would be cool as hell to see Templars SMASH people instead of just have a stupid sword everyone has lol.

We don't have enough mace roles...  :(

**Comment by WindowTalents** [comment]
Last Note:

I won't be the one who makes any of these changes, I just thought they would be worth pointing out.
